README.md

```diff
@@ -16,6 +16,8 @@ Hit `Cmd-Alt-O`/`Ctrl-Alt-O` to open the file list to the directory of the
 current file. As you edit the path the file list will automatically show
 matching files and directories. Hit `Tab` to autocomplete the path.
 
+Relative paths are considered relative to the current project's root folder.
```

Comment:
"relative to the project's _first_ root folder", if we decide not to make the other change…